models/eventAmbulances.js

```diff
@@ -10,7 +10,8 @@ module.exports = (sequelize, DataTypes) => {
     { paranoid: true }
   );
   eventAmbulances.associate = function (models) {
-    // define associations here
+    eventAmbulances.belongsTo(models.event);
+    eventAmbulances.belongsTo(models.ambulance);
```

Comment:
i'm guessing this is introduced to optimize the fetch calls - basically doing the `JOIN` stuff i mentioned previously?

Reply:
Exactly. It introduces the functionality needed to do the joins.